src/include/CmdArgsHelper.h

```diff
@@ -16,7 +16,7 @@ const char *cmdArgsStrAquarium = R"(Options and arguments:
 --discrete-gpu          : Choose discrete gpu to render the application. This is only supported on Dawn and D3D12 backend.
 --enable-alpha-blending=[0, 1] | false : Force enable alpha blending to a specific value or disable alpha blending for all models. By default, alpha blending is enabled.
 --enable-instanced-draws : specifies rendering fishes by instanced draw. By default, fishes are rendered by individual draw.Instanced rendering is only supported on dawn and d3d12 backend now.
---enable-msaa           : Enable 4 samples MSAA. MSAA of angle backend is not supported now.
+--msaa-sample-count     : specifies sample count of MSAA, while 1 for non-MSAA.
```

Comment:
Maybe `--msaa-sample-count [count]`? (and we should convert all `[]` to `<>` in the future)

Reply:
May I use the next patch to change the way we do arg parser now? I hope to introduce something like "argparse" in python.

Reply:
Sounds good, but I still hope README and the help message here can be updated.